<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="MainPageWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>801</width>
      <height>581</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 217, 159);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="MainPage">
     <widget class="QPushButton" name="CustomTripButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>330</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(83, 167, 124);</string>
      </property>
      <property name="text">
       <string>Start Custom Trip</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ASUTripButton">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>330</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(83, 167, 124);</string>
      </property>
      <property name="text">
       <string>ASU Trip</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ShortestTripButton">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>330</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(83, 167, 124);</string>
      </property>
      <property name="text">
       <string>Shortest Trip</string>
      </property>
     </widget>
     <widget class="QPushButton" name="AdminPageButton">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>330</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(83, 167, 124);</string>
      </property>
      <property name="text">
       <string>Admin Page</string>
      </property>
     </widget>
     <widget class="QLabel" name="WelcomeLabel">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>80</y>
        <width>501</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(83, 167, 124);</string>
      </property>
      <property name="text">
       <string>Welcome to College Tour Planner!</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SaddlebackTripButton">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>260</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(83, 167, 124);</string>
      </property>
      <property name="text">
       <string>SaddlebackTrip</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="StartingCollegePage">
     <widget class="QLabel" name="StartingCollegeLabel">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>621</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(83, 167, 124);</string>
      </property>
      <property name="text">
       <string>Choose Your Starting College!</string>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>169</y>
        <width>761</width>
        <height>371</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout"/>
     </widget>
     <widget class="QPushButton" name="BackTMainButton_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(83, 167, 124);</string>
      </property>
      <property name="text">
       <string>Main Page</string>
      </property>
     </widget>
     <widget class="QPushButton" name="AdminPageButton_2">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>10</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(83, 167, 124);</string>
      </property>
      <property name="text">
       <string>Admin Page</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="AdminPage">
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>240</y>
        <width>381</width>
        <height>201</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_UserName">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(83, 167, 124);</string>
           </property>
           <property name="text">
            <string>UserName</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_UserName">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(83, 167, 124);</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_Password">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(83, 167, 124);</string>
           </property>
           <property name="text">
            <string>Password</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_Password">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(83, 167, 124);</string>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Login">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(83, 167, 124);</string>
         </property>
         <property name="text">
          <string>Login</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="BackTMainButton_2">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(83, 167, 124);</string>
         </property>
         <property name="text">
          <string>Back</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>90</y>
        <width>461</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(83, 167, 124);</string>
      </property>
      <property name="text">
       <string>Login to Access the Maintenance Page</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="MaintenancePage">
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>200</y>
        <width>251</width>
        <height>131</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="AddSouvenirButton">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(83, 167, 124);</string>
         </property>
         <property name="text">
          <string>Add Souvenir</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="UploadCollegeButton">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(83, 167, 124);</string>
         </property>
         <property name="text">
          <string>Upload College From File</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>120</y>
        <width>471</width>
        <height>431</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QListWidget" name="DisplaySouvenirWidget">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(83, 167, 124);</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="MaintenancePageLabel">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>40</y>
        <width>471</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(83, 167, 124);</string>
      </property>
      <property name="text">
       <string>Welcome to the Maintenance Page</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>360</y>
        <width>160</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QPushButton" name="BackTMainButton">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(83, 167, 124);</string>
         </property>
         <property name="text">
          <string>Exit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="AdminPageButton_3">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(83, 167, 124);</string>
         </property>
         <property name="text">
          <string>Admin Page</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>806</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
